httpbp/handler_test.go

```diff
@@ -200,3 +178,84 @@ func TestHandler(t *testing.T) {
 		)
 	}
 }
+
+func TestBaseplateHandlerFactory(t *testing.T) {
+	defer func() {
+		tracing.CloseTracer()
```

Comment:
Do we need to check the error here?

Reply:
I don't think so? We don't in any other tests that do this.

Reply:
That error comes from this function, which always return nil: https://github.com/reddit/baseplate.go/blob/cbf7b6770b15e2cfd948c0ac9dcdcf8262f09ac3/mqsend/mqsend_mock.go#L27